search.php

```diff
@@ -674,22 +674,43 @@
 			eval("\$selectall = \"".$templates->get("search_threads_inlinemoderation_selectall")."\";");
 
 			$customthreadtools = '';
-			switch($db->type)
-			{
-				case "pgsql":
-				case "sqlite":
-					$query = $db->simple_select("modtools", "tid, name", "type='t' AND (','||forums||',' LIKE '%,-1,%' OR forums='')");
-					break;
-				default:
-					$query = $db->simple_select("modtools", "tid, name", "type='t' AND (CONCAT(',',forums,',') LIKE '%,-1,%' OR forums='')");
-			}
+			$forum_stats = $cache->read("forumsdisplay");
 
-			while($tool = $db->fetch_array($query))
+			if(is_moderator(0, 'canusecustomtools') && !empty($forum_stats[-1]['modtools']))
```

Comment:
https://github.com/mybb/mybb/pull/2376 should be merged before this for the `is_moderator` check to work.